packages/cli-kit/src/ui.ts

```diff
@@ -3,13 +3,40 @@ import {Input} from './ui/input'
 import {Select} from './ui/select'
 import {Bug, AbortSilent} from './error'
 import {remove, exists} from './file'
-import {info, content, token} from './output'
+import {info, content, token, logToFile} from './output'
 import {relative} from './path'
 import {isTerminalInteractive} from './environment/local'
 import {isTruthy} from './environment/utilities'
 import inquirer from 'inquirer'
+import {Listr as OriginalListr, ListrTask, ListrEvent, ListrTaskState} from 'listr2'
+
+export function newListr(tasks: ListrTask[], options?: object) {
```

Comment:
Have you considered implementing a [renderer](https://listr2.kilic.dev/renderers/default-renderer) instead? I see they have an interface you might be able to extend to add log-to-file calls.

Reply:
I originally looked into that, but these renderers are huge. It might be possible to extend an existing renderer, but I found it a big struggle to get the types right, and it wasn't worth it to me. Maybe you'd have better luck.